src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/PostgreSqlConnection.php

```diff
@@ -45,29 +43,15 @@ public function __wakeup()
         throw new \BadMethodCallException('Cannot unserialize '.__CLASS__);
     }
 
-    public function __destruct()
-    {
-        $this->unlisten();
-    }
-
-    public function reset()
-    {
-        parent::reset();
-        $this->unlisten();
-    }
```

Comment:
don't we want to keep this? we still want to unlisten on reset, don't we?

Reply:
i have reverted a portion of this. I have also verified that calling `UNLISTEN` multiple times makes no harm